gen/dcos-config-windows.yaml

```diff
@@ -32,7 +16,7 @@ package:
       {
         "role": "agent",
         "endpoint-config": "{{ windows_dcos_install_path_json }}\\etc\\dcos-diagnostics\\dcos-diagnostics-endpoint-config.json",
-        "agent-port": 1050,
+        "agent-port": 61001,
```

Comment:
that means we will not have admin router on dc/os open?
/cc @jongiddy

Reply:
it was copied from https://github.com/dcos/dcos/blob/master/gen/dcos-config.yaml#L1518

Reply:
dcos-diagnostics listens on port 1050, but changing it to 61001 would make it contact instances on other nodes through Admin Router.